src/components/Secrets.vue

```diff
@@ -34,7 +40,6 @@ export default {
 
 <style scoped>
 img{
-  width: 50px;
```

Comment:
I think I would leave this, it makes some images look a bit strange, but they all have the same dimensions, which looks cleaner in my opinion.

Reply:
Ah, fine. Some of them look a bit too stretched to me. I added 03105d9, which crops them to 50x50 instead of stretching. Is that an acceptable compromise? :)